src/omega_edit/client.ts

```diff
@@ -36,29 +37,21 @@ export function getOmegaEditPackageVersion(filePath: fs.PathLike) {
   ]
 }
 
-function getServerPidFile() {
+function getServerPidFile(port?: number) {
   return path.join(appDataPath, `serv-${serverPort}.pid`)
 }
 
-async function getOmegaEditPort(port: number | undefined = undefined) {
-  if (!port) {
-    const defaultServerPort = vscode.workspace
+async function getOmegaEditPort() {
+  if (serverPort === 0) {
+    serverPort = vscode.workspace
       .getConfiguration('dataEditor')
-      .get<number>('defaultServerPort', serverPort)
-    const portEntered = await vscode.window.showInputBox({
-      prompt: 'Enter port number to run omega-edit server on',
-      value: defaultServerPort.toString(),
-    })
-
-    if (portEntered) {
-      serverPort = parseInt(portEntered)
-    } else {
+      .get<number>('defaultServerPort', defaultServerPort)
```

Comment:
I think this variable in `dataEditor` config should be changed to `serverPort`.

So in `package.json` I believe `dataEditor.defaultServerPort` needs to be changed to `dataEditor.serverPort`. Then here it should do `.get<number>('serverPort', defaultServerPort)`. The configuration variable having `default` in the name makes me think it should not be set.